variation.go

```diff
@@ -155,6 +161,13 @@ func (g *GoFeatureFlag) AllFlagsState(user ffuser.User) flagstate.AllFlags {
 	return allFlags
 }
 
+// GetFlagsFromCache returns all the flags present in the cache with their
+// current state when calling this method. If cache hasn't been initialized, an
+// error reporting this is returned.
+func (g *GoFeatureFlag) GetFlagsFromCache() (map[string]flag.Flag, error) {
```

Comment:
Could you also expose `GetFlagsFromCache()` when using the static version of `go-feature-flag`.

As you can see with other functions, we expose them statically and within the `GoFeatureFlag` struct, you can look at `JSONVariation` for the example.